src/TLuaInterpreter.cpp

```diff
@@ -10827,6 +10827,57 @@ int TLuaInterpreter::getServerEncodingsList(lua_State* L)
     return 1;
 }
 
+// Determine operating system, moved from a linux/macos/windows only script in
+// "Other.lua" to provide better identification for "other" OSs
+int TLuaInterpreter::getOS(lua_State* L)
+{
+#if defined(Q_OS_CYGWIN)
+    // Try for this one before Q_OS_WIN32 as both are likely to be defined on
+    // a Cygwin platform
+    // CHECK: hopefully will NOT be triggered on mingw/msys
+    lua_pushstring(L, "cygwin");
+    return 1;
```

Comment:
I think you can move the `return ` outside of the `#if` for less duplication.

Reply:
Oh yeah!